libs/core/src/lib/busy-indicator/busy-indicator-extended/busy-indicator-extended.directive.ts

```diff
@@ -0,0 +1,32 @@
+import { AfterContentInit } from '@angular/core';
+import { Directive, ElementRef, Input } from '@angular/core';
+
+@Directive({
+    // tslint:disable-next-line:directive-selector
+    selector: '[fd-busy-indicator-extended]'
+})
+export class BusyIndicatorExtendedDirective implements AfterContentInit {
+    /** add extended busy indicator type */
+    @Input()
+    extended: boolean;
```

Comment:
I don't get the logic. Why are you adding an `extended` property? If you want an extended busy indicator you add the `fd-busy-indicator-extended` directive. What happens if `extended` is false? It doesn't apply the class. Isn't it the same as just omitting the directive?